test/e2e/scenarios/windows/validate-winbpf-metrics.go

```diff
@@ -0,0 +1,379 @@
+			v.KubeConfigFilePath,
+			"C:\\event-writer-helper.bat EventWriter-Curl 23.192.228.84",
+			v.NonHpcAppNamespace,
+			nonHpcLabelSelector)
+		if err != nil {
+			return err
+		}
+		numcurls--
+	}
+
+	//DROP
+	time.Sleep(20 * time.Second)
+	fmt.Printf("Produce Drop Events\n")
+	output, err = kubernetes.ExecCommandInWinPod(
+		v.KubeConfigFilePath,
+		"C:\\event-writer-helper.bat EventWriter-SetFilter -event 1 -srcIP 23.192.228.84",
+		v.EbpfXdpDeamonSetNamespace,
+		ebpfLabelSelector)
+	if err != nil {
+		return err
+	}
+
+	if strings.Contains(output, "failed") || strings.Contains(output, "error") || strings.Contains(output, "exiting") {
+		return fmt.Errorf("failed to start event writer")
+	}
+
+	numcurls = 10
+	for numcurls > 0 {
+		_, err = kubernetes.ExecCommandInWinPod(
+			v.KubeConfigFilePath,
+			"C:\\event-writer-helper.bat EventWriter-Curl 23.192.228.84",
+			v.NonHpcAppNamespace,
+			nonHpcLabelSelector)
+		if err != nil {
+			return err
+		}
+		numcurls--
+	}
+
+	fmt.Println("Waiting for basic metrics to be updated as part of next polling cycle")
+	time.Sleep(60 * time.Second)
+	promOutput, err = v.GetPromMetrics()
+	if err != nil {
+		return err
+	}
+
+	//TBR
+	fmt.Println(promOutput)
+
+	postTestFwdCount, err := prom.GetMetricGuageValueFromBuffer([]byte(promOutput), "networkobservability_forward_count", fwd_labels)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Post test - networkobservability_forward_count value %f, labels: %v\n", preTestFwdBytes, fwd_labels)
+
+	postTestFwdBytes, err := prom.GetMetricGuageValueFromBuffer([]byte(promOutput), "networkobservability_forward_bytes", fwd_labels)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Post test - networkobservability_forward_bytes value %f, labels: %v\n", postTestFwdBytes, fwd_labels)
+
+	postTestDrpBytes, err := prom.GetMetricGuageValueFromBuffer([]byte(promOutput), "networkobservability_drop_bytes", drp_labels)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Post test - networkobservability_drop_bytes value %f, labels: %v\n", postTestDrpBytes, drp_labels)
+
+	postTestDrpCount, err := prom.GetMetricGuageValueFromBuffer([]byte(promOutput), "networkobservability_drop_count", drp_labels)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Post test - networkobservability_drop_count value %f, labels: %v\n", preTestDrpBytes, drp_labels)
+
+	if postTestFwdBytes <= preTestFwdBytes {
+		return fmt.Errorf("networkobservability_forward_bytes not incremented")
+	}
+
+	if postTestDrpBytes <= preTestDrpBytes {
+		return fmt.Errorf("networkobservability_drop_bytes not incremented")
+	}
+
+	if postTestFwdCount <= preTestFwdCount {
+		return fmt.Errorf("networkobservability_forward_count not incremented")
+	}
+	if postTestDrpCount <= preTestDrpCount {
+		return fmt.Errorf("networkobservability_drop_count not incremnted")
+	}
+
+	// Advanced Metrics
+	adv_fwd_count_labels := map[string]string{
+		"direction":     "egress",
+		"ip":            "23.192.228.84",
```

Comment:
what is this hardcoded IP? Can we make it configurable?

Reply:
Its repeated many times. Make it a constant at least if it is hard to specify it in test config